src/streamingDelegate.ts

```diff
@@ -359,6 +434,15 @@ export class StreamingDelegate implements CameraStreamingDelegate {
 
       let ffmpegArgs = this.videoConfig.source!;
 
+      if (this.sipConfig) {
+        ffmpegArgs += // SIP audio via RTP
+        '-hide_banner' +
```

Comment:
```suggestion
        ' -hide_banner' +
```

This was causing an issue with my FFmpeg stream since it was appending the `-hide_banner`directly to the input URL without a space as a separate argument.

For example my config was

```
"source": "-i rtsp://192.168.1.3/h264_stream",
```

and the FFmpeg command was executed as

```
ffmpeg.exe -i rtsp://192.168.1.3/h264_stream-hide_banner -protocol_whitelist pipe,udp,rtp,file,crypto...
```

which leads to an error since `rtsp://192.168.1.3/h264_stream-hide_banner` is not the correct stream URL.